designdocs/provider_contexts.md

```diff
@@ -0,0 +1,231 @@
+# Provider Contexts Design Doc
+
+## Context and Scope
+
+On Frequency, a Provider may either represent an application or a company.
+This document will outline the changes that will allow a Provider to represent a company with one or multiple applications.
+It also enables social wallet providers, custodial and non-custodial, to provide safe application information to their users.
+
+This is an expansion of the [Registered Provider](https://github.com/frequency-chain/frequency/blob/main/designdocs/provider_registration.md) concepts.
+
+Reference Document: [SIWF Specification](https://projectlibertylabs.github.io/siwf/v2/docs/)
+
+### Why Is This Needed?
+
+#### Concept of an Application Separate from a Provider
+
+Many companies have a single major product, but some companies have multiple applications that serve different needs.
+While each application could be a separate provider, this causes two issues: The sharing of capacity and the user permissions.
+
+Multiple providers require separate Capacity staking.
+As these applications would be from the same company, it is reasonable to say that they should share Capacity.
+Instead of building a Capacity sharing system, it is more reasonable to merely be the same Provider and provide a system of application identification.
```

Comment:
```suggestion
```suggestion
Having separate providers would require a single company to have separate Capacity staking for each application.
As each these applications are still part of the same company, it is reasonable to say that those applications should share Capacity.
Instead of building a Capacity sharing system which has different trade-offs, it is reasonable for those applications to use the same Provider and provide a system of application identification that is separate from provider identification for these cases..
```
```